docs/addons/karpenter.md

```diff
@@ -38,37 +40,32 @@ const karpenterAddOn = new blueprints.addons.KarpenterAddOn({
       annotations: {
           "eks-blueprints/owner": "young"
       },
-      taints: [{
-          key: "workload",
-          value: "test",
-          effect: "NoSchedule",
-      }],
       requirements: [
-          { key: 'node.kubernetes.io/instance-type', operator: 'In', values: ['m5.2xlarge'] },
-          { key: 'topology.kubernetes.io/zone', operator: 'In', values: ['us-west-2a','us-west-2b', 'us-west-2c']},
+          { key: 'node.kubernetes.io/instance-type', operator: 'In', values: ['m5.large'] },
+          { key: 'topology.kubernetes.io/zone', operator: 'In', values: [`${region}a`, `${region}b`, `${region}c`] },
           { key: 'kubernetes.io/arch', operator: 'In', values: ['amd64','arm64']},
-          { key: 'karpenter.sh/capacity-type', operator: 'In', values: ['spot']},
+          { key: 'karpenter.sh/capacity-type', operator: 'In', values: ['on-demand']},
```

Comment:
good one, let's add a comment here that they can use spot if need to. Very handy for cost saving for some folks.